Healthchecks.md

```diff
@@ -1,6 +1,6 @@
 # S3 Healthcheck
 
-Scality S3 exposes a healthcheck route `/_/healthcheck` which returns a
+Scality S3 exposes a healthcheck route `/healthcheck` which returns a
```

Comment:
should indicate this is not on the same port as s3, actually.

Reply:
actually not, in the ticket it is defined that we put it on a different port, it should be on the metrics port, but sure we should indicate the port to use.

Reply:
yes exactly my point : looking at it here we may think that healthcheck is on the same port as S3, which is not the case. So we should indicate it is on this metrics port (or admin, not sure how it is called in the config).

Reply:
will do.